core/domain/stats_domain.py

```diff
@@ -48,19 +59,23 @@ def get_multi_by_multi_explorations(
         """Gets domain objects given a list of exploration and state tuples.
 
         Args:
-            exploration_state_list: a list of exploration ID and state name
-                tuples
-            rule_str_list: a list of rule spec strings which are used to filter
-                the answers matched to the provided explorations and states
-
-        Returns a list of StateRuleAnswerLog objects containing answers matched
-        to each exploration ID-state name. The number of elements returned is
-        the same as the number of elements passed in exploration_state_list. The
-        number of answers in each StateRuleAnswerLog object depends on the
-        number of answers matched against each rule spec strings specified in
-        rule_str_list.
+            exploration_state_list: list(tuple). A list of exploration ID and
+                state name tuples.
+            rule_str_list: list. A list of rule spec strings which are used to
```

Comment:
I did research this a bit, and I guess the description given should do fine. The tests in stats_domain_test.py helped me a lot.